addons/uom/data/uom_data.xml

```diff
@@ -139,6 +150,14 @@
         <field name="uom_type">bigger</field>
     </record>
 
+    <!-- SURFACE -->
+    <record id="uom_square_foot" model="uom.uom">
+        <field name="name">ft²</field>
```

Comment:
Do you think this one should also be switchable in the settings ? 
![image](https://user-images.githubusercontent.com/12071695/112183747-d5366d00-8bfe-11eb-87c5-50ecbe8ef61b.png)

Reply:
I don't think as we don't use those surfaces anywhere for now.